edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -170,7 +174,8 @@ def _decode_and_verify_token(token, jwt_issuer):
     token_issuer = decoded_token.get('iss')
     issuer_matched = any(issuer['ISSUER'] == token_issuer for issuer in get_jwt_issuers())
     if not issuer_matched:
-        logger.info('Token decode failed due to mismatched issuer [%s]', token_issuer)
+        if not suppress_logging:
+            logger.info('Token decode failed due to mismatched issuer [%s]', token_issuer)
         raise jwt.InvalidTokenError('%s is not a valid issuer.', token_issuer)
```

Comment:
Reviewers: Codecov is reporting that we don't have coverage for an error with the invalid issuer.  I could add a test for that.  

Codecov seems to be happy with coverage of the rest, even though we aren't testing `suppress_logging = True`, but I'm happy with that.  I wasn't intending to add testing here anyway.  I did fix the test in ecommerce that wants this logging to be suppressed.

Reply:
No longer applies.